code/game/machinery/computer/arena.dm

```diff
@@ -0,0 +1,420 @@
+#define ARENA_RED_TEAM "red"
+#define ARENA_GREEN_TEAM "green"
+#define ARENA_DEFAULT_ID "arena_default"
+#define ARENA_CORNER_A "cornerA"
+#define ARENA_CORNER_B "cornerB"
+
+/// Arena related landmarks
+/obj/effect/landmark/arena
+	name = "arena landmark"
+	var/landmark_tag
+	var/arena_id = ARENA_DEFAULT_ID
+
+/obj/effect/landmark/arena/start
+	name = "arena corner A"
+	landmark_tag = ARENA_CORNER_A
+
+/obj/effect/landmark/arena/end
+	name = "arena corner B"
+	landmark_tag = ARENA_CORNER_B
+
+/// Controller for admin event arenas
+/obj/machinery/computer/arena
+	name = "arena controller"
+	/// Arena ID
+	var/arena_id = ARENA_DEFAULT_ID
+	/// Enables/disables spawning
+	var/ready_to_spawn = FALSE
+	/// Assoc list of map templates indexed by user friendly names
+	var/static/list/arena_templates = list()
+	/// Were the config directory arenas loaded
+	var/static/default_arenas_loaded = FALSE
+	/// Name of currently loaded template
+	var/current_arena_template = "None"
+	// What turf arena clears to
+	var/empty_turf_type = /turf/open/indestructible
+	// List of team ids
+	var/list/teams = list(ARENA_RED_TEAM,ARENA_GREEN_TEAM)
+	/// List of hud instances indedxed by team id
+	var/static/list/team_huds = list()
+	/// List of hud colors indexed by team id
+	var/static/list/team_colors = list(ARENA_RED_TEAM = "red", ARENA_GREEN_TEAM = "green")
```

Comment:
I don't get the point of this list. They're both always the same value. Also this list isnt used.

Reply:
Oh whoops i added table but forgot to use it.